src/Microsoft.ServiceFabric.Actors/Runtime/ActorStateManager.cs

```diff
@@ -212,7 +212,7 @@ internal ActorStateManager(ActorBase actor, IActorStateProvider actorStateProvid
             {
                 var stateMetadata = this.stateChangeTracker[stateName];
 
-                // Check if the property was marked as remove in the cache
+                // Checks if the property was marked as remove in the cache.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.